### PR TITLE
adds test for token list fetching

### PR DESCRIPTION
**Pull request by schmanu**
Just a small test case which would have found the error while fetching / parsing the token list.

### CODE REVIEW

**Review by bh2smith — APPROVED** [review]

File: src/__tests__/tokenList.test.ts [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
+  });
+
+  it("Should not crash on errors", async () => {
+    jest.spyOn(window, "fetch").mockImplementation(() => {
+      return Promise.reject("Unexpected error.");
+    });
+
+    const resultingTokens = await fetchTokenList(1);
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    expect(resultingTokens).to.be.empty;
```
Comment (bh2smith) [inline src/__tests__/tokenList.test.ts:56]: Could you avoid the lint ignore by putting the await statement directly into the expect?

**Review by bh2smith — APPROVED** [review]
Looks good. Love tests. One suggestion to remove lint ignores.

File: src/__tests__/tokenList.test.ts (hunk 1 above)
Comment (schmanu) [reply src/__tests__/tokenList.test.ts:56]: The problem here is that the to.be.empty is not a function so it looks like a normal expression which is not used.
But in fact this actually does something (checking if the object ist empty). There are other tests with the same issue.

I found a more elegant solution which disabled this rule for all test files by including this in the eslint config:

```
{
    "overrides": [
        {
            "files": ["*.test.ts", "*.test.tsx"],
            "rules": {
                "no-unused-expressions": "off"
            }
        }
    ]
}

```

Comment (bh2smith) [reply src/__tests__/tokenList.test.ts:56]: Seems legit... go for it!

**Comment by bh2smith** [comment]
Looks ready!

**Pull request merged**